flow-shop/chapter.md

```diff
@@ -248,8 +249,9 @@ To make locating the correct instance easier, we assume that every line will be
             sys.exit(0)
 ```
 
-Finally, we parse the data directly -- converting each of the tasks processing times to an integer and storing them in a list. As a final step, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above (i.e., every item in data should correspond to a particular job).
+Finally, we parse the data directly, converting each of the tasks' processing times to an integer and storing them in a list. Finally, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above. (That is, every item in `data` should correspond to a particular job.)
```

Comment:
Well the original each refers to the times, so I think your correction was the right one. My rephrasing just gets rid of the s' -- I realize it's proper English, but it's a royal pain to pronounce (and hence I shy away from it ;)).

Your most recent suggestion looks good to me!